msg/NeuralControl.msg

```diff
@@ -0,0 +1,9 @@
+# Neural control message
+
```

Comment:
Is there more to say about this? If so, can you add a long description? See https://github.com/PX4/PX4-Autopilot/blob/main/msg/versioned/BatteryStatus.msg for inspiration, but basically if this has particular publishers and subscribers, or there is anything that isn't implied by the name, a good place to help people understand where they look for more information.